extensions/entry-extension-collapsible/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Add to `.gitignore`?

Reply:
After googling seems like this file causes issues sometimes (although I didn't have any in this case), so make sense to put it in `.gitignore`. Will do it!